components/coding/profileV2/achievement_components/AchievementComponent.tsx

```diff
@@ -113,6 +124,22 @@ const AchievementComponent = ({
           })}
         </div>
       )}
+      <div className="flex place-content-between space-x-4 px-4">
+        <Button
+          label={"Save"}
+          onClick={() => handleOnSaveButtonClick()}
+        ></Button>
+        <button
+          onClick={() => setEditMode(!editMode)}
+          className="w-5 h-5 cursor-pointer hover:text-yellow-600"
+        >
+          {editMode ? (
```

Comment:
instead of using the ternary this way I would do something like this

`<PencilAltIcon className={'w-5 h-5 cursor-pointer ${editMode ? "text-yellow-600" : "hover:text-yellow-600"} '} />`

use backticks instead of single quotes though. Thing that would be cleaner.

Reply:
Will do! thank you for the feedback